1.3/main.d.ts

```diff
@@ -973,13 +973,18 @@ declare module Meteor {
   var isServer: boolean;
   var isProduction: boolean;
   var release: string;
-  var settings: {
+  /** Global props **/
+  
+  /** Settings **/
+  interface Settings {
```

Comment:
@barbatus My mistake, wasn't really paying enough attention when I went to make this change! I assume you would like me to run the build script before pushing to this PR?